pkg/api/v1/types.go

```diff
@@ -2196,6 +2208,9 @@ type PodStatus struct {
 	// More info: http://kubernetes.io/docs/user-guide/pod-states#container-statuses
 	// +optional
 	ContainerStatuses []ContainerStatus `json:"containerStatuses,omitempty" protobuf:"bytes,8,rep,name=containerStatuses"`
+	// QOS class of the Pod
```

Comment:
I should have caught this early, but please provide more description of what this field is.  It's not a very good description.